CODE_OF_CONDUCT.md

```diff
@@ -19,7 +19,7 @@ Examples of unacceptable behavior by participants include:
 * The use of sexualized language or imagery and unwelcome sexual attention or advances
 * Trolling, insulting/derogatory comments, and personal or political attacks
 * Public or private harassment
-* Publishing others' private information, such as a physical or electronic address, without explicit permission
+* Publishing other's private information, such as a physical or electronic address, without explicit permission
```

Comment:
This is not grammatically correct. others' is rightly used already. 

others' stands for a plural usage, which is correct already.